src/System.Runtime/tests/System/StringTests.cs

```diff
@@ -1618,23 +1618,24 @@ public static IEnumerable<object[]> Join_ObjectArray_TestData()
             yield return new object[] { null, new object[] { "Foo", "Bar", "Baz" }, "FooBarBaz" };
             yield return new object[] { "$$", new object[] { "Foo", null, "Baz" }, "Foo$$$$Baz" };
 
-            // Join does nothing if array[0] is null
-            yield return new object[] { "$$", new object[] { null, "Bar", "Baz" }, "" };
+            // Test join when first value is null
+            yield return new object[] { "$$", new object[] { null, "Bar", "Baz" }, "$$Bar$$Baz" };
 
             // Join should ignore objects that have a null ToString() value
             yield return new object[] { "|", new object[] { new ObjectWithNullToString(), "Foo", new ObjectWithNullToString(), "Bar", new ObjectWithNullToString() }, "|Foo||Bar|" };
         }
 
-        [ActiveIssue(13747)]
         [Theory]
         [MemberData(nameof(Join_ObjectArray_TestData))]
         public static void Join_ObjectArray(string separator, object[] values, string expected)
```

Comment:
I think the right approach is to split this in 2 methods.

The first method will test the new behavior and you will have to add these 2 attributes to make sure it does not run on the previous versions of the platform:
```csharp
[SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
[SkipOnTargetFramework(TargetFrameworkMonikers.Netcoreapp1_0)]
```

And another method will test the old behavior. For that you will have to add this attribute:
```csharp
[SkipOnTargetFramework(TargetFrameworkMonikers.Netcoreapp1_1)]
```
To make sure it is not running on the new version of the platform.

As I mentioned in the other PR, the netstandard17 define is actually set for more build configuration that you need it.